source/extensions/tracers/xray/daemon.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+// The structures in this file are merely for JSON serialization to communicate with the X-Ray
+// daemon. The X-Ray daemon does not speak protobuf.
+// TODO(marcomagdy): is there anyway to prevent protoc from generating types for java/C#/etc.?
```

Comment:
What do you mean here? We shouldn't be generating any Java/C#/etc. code for C++.